### PR TITLE
Got rid of class Configurable

### PR DESCRIPTION
**Pull request by SlNPacifist**
ClusterProcess now inherits from EventEmitterEx
Added tests for helper functions (has, get, set) of Configuration

### CODE REVIEW

**Comment by kaero** [comment]
\cc @Flackus

File: lib/cluster_process.js [hunk 1]
```diff
@@ -23,9 +25,9 @@ function has(context, propName) {
 /**
  * @constructor
  * @class ClusterProcess
- * @augments Configurable
+ * @augments EventEmitterEx
```
Comment (Flackus) [inline lib/cluster_process.js]: 👍

File: lib/cluster_process.js [hunk 2]
```diff
@@ -65,6 +73,46 @@ Object.defineProperty(ClusterProcess.prototype, 'isWorker', {
...
+ * @param {Object} config
+ * @param {Boolean} [applyEnv=true]
+ * @param {String} [basedir=process.cwd()] for Configuration#resolve relative paths
+ * @returns {ClusterProcess} this
+ * @throws {LusterConfigurationError} if configuration check failed (check errors will be logged to STDERR)
+ * @public
+ */
+ClusterProcess.prototype.configure = function(config, applyEnv, basedir) {
+    if (typeof applyEnv === 'undefined' || applyEnv) {
+        // @todo may be create a deep copy and do not modify origin?
```
Comment (Flackus) [inline lib/cluster_process.js]: @kaero may be now is the moment to fix it or remove todo?

File: test/unit/test/cluster_process.js [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+    TestClusterProcess;
+
+/**
+ * ClusterProcess is an abstract class, it cannot be instantiated for tests;
+ * TestClusterProcess is a smalles possible descendant of ClusterProcess;
+ */
+TestClusterProcess = ClusterProcess.create(function TestClusterProcess() {});
+TestClusterProcess.prototype._setupIPCMessagesHandler = function() {};
+
+
```
Comment (Flackus) [inline test/unit/test/cluster_process.js]: @SlNPacifist can you please add an eslint rule for no multiple line breaks?

Comment (Flackus) [inline test/unit/test/cluster_process.js]: though it can be done later

File: lib/cluster_process.js (hunk 2 above)
Comment (Flackus) [inline lib/cluster_process.js]: guys?

Comment (SlNPacifist) [inline lib/cluster_process.js]: Looks like we can just drop this todo

Comment (Flackus) [inline lib/cluster_process.js]: ok, let's drop it.

**Pull request merged**